pyproject.toml

```diff
@@ -184,6 +186,7 @@ forecasting = [
   'statsforecast<1.7,>=0.5.2; python_version < "3.12"',
   "statsmodels<0.15,>=0.12.1",
   'tbats<1.2,>=1.1; python_version < "3.12"',
+  'autots',
```

Comment:
Can you please add lower/upper bounds? Plus, please order alphabetically as @fkiraly suggested above.